### DISCUSSION TITLE
Does RGB565 support exist in ImageSharp?

### DISCUSSION THREAD

**Discussion by ThinkLoog** [SixLabors/ImageSharp#1826] [Q&A] [answered]
Does ImageSharp have support for RGB565 colours?

**Comment by brianpopow** [comment, marked as answer]
No, the ImageSharp main project does not have RGB565, but i have implemented this format for the [ImageSharp.Texture](https://github.com/SixLabors/ImageSharp.Textures) project. We have released a early alpha version `ImageSharp.Texture` which you could use (from our [MyGet feed](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.ImageSharp.Textures)), but the project is still in an very early stage and i would not recommend using this in production.

You could alternatively just copy the `RGB565` source from there to your project and use it that way.

**Reply by ThinkLoog** [reply]
I'll look into copying the RGB565 part from the ImageSharp.Texture project. Thank you!
